lib/base/Modeler.js

```diff
@@ -9,10 +9,6 @@ import alignToOriginModule from '@bpmn-io/align-to-origin';
 
 import executableFixModule from 'bpmn-js-executable-fix';
 
-import signavioCompatModule from 'bpmn-js-signavio-compat';
```

Comment:
Don't we compat signavio anymore 🙁 ?

Reply:
We are still compatible with signavio, but don't need the extra module for it.
We now handle Subprocesses similarly to signavio and bpmn-js can expand them out of the box. Happy yo show you what we are doing :slightly_smiling_face:

Reply:
Collapse/Expand of multi-plane diagrams is now part of the core modeling experience

Reply:
Oh nice, I didn't know signavio compat was only about sub-processes collapsing 👍

Reply:
Being able to remove the compatibility plugin is a good sign. 👍🏻